FlowCrypt/Controllers/Compose/Extensions/ComposeViewController+ActionHandling.swift

```diff
@@ -22,7 +22,7 @@ extension ComposeViewController {
                                     .unique()
                                     .sorted()
 
-                updateState(with: .searchEmails(recipients))
+                updateView(with: .searchEmails(recipients))
```

Comment:
Could it be `updateView(newState: .searchEmails(recipients))` ? that would make it clearer. These `with:`,  `for:` etc keywords often obscure what the thing is if the variable doesn't make it obvious.